docker/esri/docker-compose.yml

```diff
@@ -29,14 +29,16 @@
 
 services:
   pygeoapi:
+    env_file:
```

Comment:
@KilimAnnejaro you can remove these lines from here, since docker compose reads `.env` by default. No need to specify an `env_file`.